components/member-profile/contribution/index.js

```diff
@@ -8,13 +9,70 @@ const renderPRLinks = (prList) =>
     return <PRLink link={url} key={index} />;
   });
 
+const memberSiteHostName = 'members.realdevsquad.com';
```

Comment:
constants should be capitalized

`HOST_NAME`
`membersSiteHostName` makes less sense, because we are already accessing members, so we need not explicitly say `membersSite`

Can you create a new file `appConstants` at the root and put the constants there and import it here?